app/scripts/controllers/transactions/lib/util.js

```diff
@@ -1,8 +1,8 @@
 import { addHexPrefix, isValidAddress } from 'ethereumjs-util'
 
 const normalizers = {
-  from: (from, lowerCase = true) => (lowerCase ? addHexPrefix(from).toLowerCase() : addHexPrefix(from)),
-  to: (to, lowerCase = true) => (lowerCase ? addHexPrefix(to).toLowerCase() : addHexPrefix(to)),
+  from: (from, lowerCase) => (lowerCase ? addHexPrefix(from).toLowerCase() : addHexPrefix(from)),
```

Comment:
This is not quite what I meant; this `doNotNormalize` parameter is overkill. We don't need to conditionally skip the `toLowerCase` step for the `from` parameter - we can just remove the `toLowerCase` stuff altogether. It's never true - we never want that.